packages/react/src/infrastructure/helio-api/HelioApiAdapter.ts

```diff
@@ -123,4 +124,48 @@ export const HelioApiAdapter = {
     ).json();
     return currencies || [];
   },
+
+  async getTokenSwapMintAddresses(
```

Comment:
Aren't you using HelioApiAdapterV1 ? I would prefer to move all from here to there and remove this file.

Reply:
we are also deprecating all the old endpoints and going to remove those

Reply:
sure. that needs to be done during refactoring. we don't need for 2 adapters for sure